pkgs/servers/xmpp/ejabberd/default.nix

```diff
@@ -1,23 +1,248 @@
-{stdenv, fetchurl, expat, erlang, zlib, openssl, pam, lib}:
+{ stdenv, writeScriptBin, lib, fetchurl, fetchgit
+, erlang, openssl, expat, libyaml, bash, gnused, gnugrep, coreutils, utillinux, procps
+, withMysql ? false
+, withPgsql ? false
+, withSqlite ? false, sqlite
+, withPam ? false, pam
+, withZlib ? true, zlib
+, withRiak ? false
+, withElixir ? false, elixir
+, withIconv ? true
+, withLager ? true
+, withTools ? false
+, withRedis ? false
+}:
 
-stdenv.mkDerivation rec {
-  version = "2.1.13";
+let
+  ctlpath = lib.makeSearchPath "bin" [ bash gnused gnugrep coreutils utillinux procps ];
+
+  fakegit = writeScriptBin "git" ''
+    #! ${stdenv.shell}
+    exit 0
+  '';
+
+  # These can be extracted from `rebar.config.script`
+  # Some dependencies are from another packages. Try commenting them out; then during build
+  # you'll get necessary revision information.
+  ejdeps = {
+    p1_cache_tab = fetchgit {
```

Comment:
I'd make all of these into `fetchFromGitHub`. It's quite a bit more efficient than `fetchgit` when using GitHub.